src/analytics/index.ts

```diff
@@ -14,23 +14,22 @@ function isInternalFlag(email: string, isInternal = false) {
   return '';
 }
 
-function getUrl(type?: string) {
+function getUrl(type?: string, isPreview = false) {
```

Comment:
@Hyperkid123 should we place the `isPreview` parameter before `type` as it is not technically optional (even though we have the default value)? 

This would prevent us from having to specify `getUrl(undefined, isPreview)`.

Another alternative would be to convert `getUrl` to accept an object so the key ordering doesn't matter.